src/NpmUtilities.js

```diff
@@ -134,6 +134,11 @@ export default class NpmUtilities {
   }
 
   static getTagOpts(registry) {
-    return registry ? {env: {npm_config_registry: registry}} : null;
+    const opts = {};
+    if (registry) {
+      opts.env = Object.assign({}, process.env, {npm_config_registry: registry});
```

Comment:
That's totally the right thing to do, since _any_ non-`null` value overwrites everything, and we definitely want to transfer everything else.